docs/sources/setup/install/helm/install-scalable/aws.md

```diff
@@ -0,0 +1,578 @@
+  - The `ruler` section defines the configuration for the ruler, including the S3 bucket region and name. This tells Loki where to store the alert and recording rules.
+  - For the full Loki configuration, refer to the [Loki Configuration]({{< relref "../../../../configure" >}}) documentation.
+
+- **Storage:**
+  - Defines where the Helm chart stores data.
+  - Set the type to `s3` since we are using Amazon S3.
+  - Configure the bucket names for the chunks and ruler to match the buckets created earlier.
+  - The `s3` section specifies the region of the bucket.
+
+- **Service Account:**
+  - The `serviceAccount` section is used to define the IAM role for the Loki service account.
+  - This is where the IAM role created earlier is linked.
+
+- **Gateway:**
+  - Defines how the Loki gateway will be exposed.
+  - We are using a `LoadBalancer` service type in this configuration.
+
+
+### Deploy Loki
+
+Now that you have created the `values.yaml` file, you can deploy Loki using the Helm chart.
+
+1. Deploy using the newly created `values.yaml` file:
+
+    ```bash
+    helm install --values values.yaml loki grafana/loki -n loki --create-namespace
+    ```
+    **It is important to create a namespace called `loki` as our trust policy is set to allow the IAM role to be used by the `loki` service account in the `loki` namespace. This is configurable but make sure to update your service account**
+
+1. Verify the deployment:
+
+    ```bash
+    kubectl get pods -n loki
+    ```
+    You should see the Loki pods running.
+    ```console
+    NAME                           READY   STATUS    RESTARTS   AGE
+    loki-backend-0                 2/2     Running   0          43m
+    loki-backend-1                 2/2     Running   0          43m
+    loki-canary-c7hbw              1/1     Running   0          43m
+    loki-canary-wtxb8              1/1     Running   0          43m
+    loki-chunks-cache-0            2/2     Running   0          43m
+    loki-gateway-844ff6d84-9zg5k   1/1     Running   0          43m
+    loki-read-848c686c9d-kd9xt     1/1     Running   0          43m
+    loki-read-848c686c9d-vx697     1/1     Running   0          43m
+    loki-results-cache-0           2/2     Running   0          43m
+    loki-write-0                   1/1     Running   0          43m
+    loki-write-1                   1/1     Running   0          43m
+    loki-write-2                   1/1     Running   0          43m
+    ```
+  
+### Find the Loki Gateway Service
+
+The Loki Gateway service is a LoadBalancer service that exposes the Loki gateway to the internet. This is where you will write logs to and query logs from. By default NGINX is used as the gateway.
+
+{{< admonition type="caution" >}}
+The Loki Gateway service is exposed to the internet. It is recommended to secure the gateway with authentication. Refer to the [Authentication]({{< relref "../../../../operations/authentication" >}}) documentation for more information.
```

Comment:
passive voice.  "We recommend" to secure. 

https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#write-in-active-voice